ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -126,9 +132,24 @@ objects:
 
         # test specific code
         # don't let clients impact the global kubeconfig
-        mkdir -p /go/src/github.com/openshift/
-        cp -r /tmp/shared/openshift-azure /go/src/github.com/openshift/
-        cd /go/src/github.com/openshift/openshift-azure/
+        if [[ "${TEST_IMAGE}" == *"/bushslicer"* ]]; then
+          TEST_NODE_OPTS="etcd:master:node"
+          TEST_CLUSTER="${RESOURCEGROUP}.eastus.cloudapp.azure.com"
+          export TESTENV="BUSHSLICER_V3"
+          export BUSHSLICER_DEFAULT_ENVIRONMENT=online_starter
+          export BUSHSLICER_DEBUG_AFTER_FAIL=1
+          export OPENSHIFT_ENV_ONLINE_STARTER_HOSTS="${TEST_CLUSTER}:${TEST_NODE_OPTS}"
+          # login to cluster as customer-cluster-admin
+          PASSWD=$(awk '/customerAdminPasswd:/{ print $2 }' /tmp/shared/openshift-azure/_data/containerservice.yaml)
```

Comment:
LIkewise, if this is going to be a standalone script, expect PASSWD to be set in the script.